src/openrct2/network/Network.cpp

```diff
@@ -1018,7 +1024,7 @@ void Network::Client_Send_AUTH(const char* name, const char* password, const cha
 {
     std::unique_ptr<NetworkPacket> packet(NetworkPacket::Allocate());
     *packet << (uint32)NETWORK_COMMAND_AUTH;
-    packet->WriteString(NETWORK_STREAM_ID);
+    packet->WriteString(network_get_version().c_str());
```

Comment:
Why is this necessary? Can't you keep using NETWORK_STREAM_ID here?

Reply:
I'd rather expose and use a single interface everywhere

Reply:
Oh, alright then. This will need to be rebased though, and the network version has increased in the meantime (from 31 to 32).

Reply:
> Edit: rebased it.

please don't edit the comment to say so, make another one. It's easy to miss such an edit.